HiP-DataStore.Model/Entity/Rating.cs

```diff
@@ -0,0 +1,32 @@
+﻿using PaderbornUniversity.SILab.Hip.DataStore.Model.Events;
+using System;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Entity
+{
+    public class Rating
```

Comment:
Note that we are no longer querying the Mongo database for rating-related things (since it's all in RAM)... so we don't even need to store ratings in the Mongo database, right?